h2o-algos/src/main/java/hex/ensemble/StackedEnsemble.java

```diff
@@ -140,11 +140,22 @@ private Frame prepareTrainingLevelOneFrame(StackedEnsembleModel.StackedEnsembleP
           throw new H2OIllegalArgumentException("Failed to find the xval predictions frame id. . .  Looks like keep_cross_validation_predictions wasn't set when building the models.");
 
         Frame aFrame = DKV.getGet(aModel._output._cross_validation_holdout_predictions_frame_id);
+
         if (null == aFrame)
           throw new H2OIllegalArgumentException("Failed to find the xval predictions frame. . .  Looks like keep_cross_validation_predictions wasn't set when building the models, or the frame was deleted.");
 
         baseModels.add(aModel);
-        baseModelPredictions.add(aFrame);
+        if(!aModel._output.isMultinomialClassifier()){
+            baseModelPredictions.add(aFrame);
+        }else {
+            //Create temp `multFrame` that will hold multinomial class predictions to avoid mutating original
+            //class predictions from the model
+            Frame multFrame = new Frame(aFrame);
+            DKV.put(multFrame);
```

Comment:
I don't think you have to install the temporary frame to DKV

Reply:
Yes, you are correct. Technically I can just do this:
` baseModelPredictions.add(aFrame.subframe(1,aFrame.numCols()));` since the first column is what I want to drop, i.e., `predict`. Does that make sense?

Reply:
Of course the above is assuming the frame layout will not change :)

Reply:
This is possible too:
`          List<String> predColNames= new ArrayList<>(Arrays.asList(aFrame.names()));
            predColNames.remove("predict");
            String[] multClassNames  = predColNames.toArray(new String[0]);
            baseModelPredictions.add(aFrame.subframe(multClassNames));`